guardian/secure-contact
# Remove PGP key directory-related components

## What does this change?

We're separating the PGP key directory listing out to its own service (see https://github.com/guardian/pgp-directory/pull/1).

As a result, we can remove related components etc from this repo, so that's what we're doing.

Both services used some shared resources (page template, icon, css etc) so for now those are duplicated across both repos. In future we may be able to do something better that that but in the meantime, let's just get it running.

Anything that was exclusively in support of the PGP directory service has been moved over.

## How to test

Try to update the cloudformation and deploy this (from the branch perhaps?) and see how it goes. 

## How can we measure success?

If the secure contact service is running correctly after the deploy, success!

## Have we considered potential risks?

I may have removed something I shouldn't, but in that scenario I'd expect to see a failure during the cloudformation update and/or deploy.

It's possible that this may work but there are latent issues in the `pgp-directory` service which may result in linking to stale or broken resources from the secure contact service.

## Images

N/A

## Accessibility



-   [x] None of the above (there are no UI changes here)
